### PR TITLE
Update issue workflow

### PR DESCRIPTION
**Pull request by allisonport-db**
Tested on my fork

### CODE REVIEW

File: .github/workflows/new_updated_issue.yaml [hunk 1]
```diff
@@ -9,11 +13,22 @@ on:
...
-          column: Needs Review
+          column: Needs Review (ENG)
           repo-token: ${{ secrets.PROJECT_BOARD_AUTOMATION_TOKEN }}
+      - name: Move to Needs Review
+        if: ${{ github.event.issue.assignee == null || contains(env.devrel_usernames, github.event.issue.assignee.login) }}
+        uses: alex-page/github-project-automation-plus@v0.8.1
+        with:
+          project: oss-delta-issues
+          column: Needs Review
+          repo-token: ${{ secrets.PROJECT_BOARD_AUTOMATION_TOKEN }}
```
Comment (zsxwing) [inline .github/workflows/new_updated_issue.yaml:34]: nit: missing a new line

**Review by zsxwing — APPROVED** [review]

File: .github/workflows/new_updated_issue.yaml [hunk 2]
```diff
@@ -9,10 +13,20 @@ on:
 
 jobs:
   automate-new-updated-issues:
-    if: ${{ !github.event.issue.pull_request && !contains('allisonport-db dennyglee scottsand-db tdas vkorukanti zsxwing', github.event.sender.login) }}
+    if: ${{ !github.event.issue.pull_request &&
```
Comment (allisonport-db) [inline .github/workflows/new_updated_issue.yaml:16]: test

File: .github/workflows/new_updated_issue.yaml [hunk 3]
```diff
@@ -1,5 +1,9 @@
 name: Add new and updated issues to Needs Review
 
+env:
```
Comment (zsxwing) [inline .github/workflows/new_updated_issue.yaml:3]: remove this?

Comment (allisonport-db) [reply .github/workflows/new_updated_issue.yaml:3]: we still use it for determining which column to move to (lines 21 and 28). I will remove it from the updated_pull_request file however :)

Comment (allisonport-db) [reply .github/workflows/new_updated_issue.yaml:3]: updated below

**Review by zsxwing — APPROVED** [review]

**Pull request closed without merging**